src/overlays/actors/ovl_En_Si/z_en_si.c

```diff
@@ -17,8 +19,9 @@ void EnSi_Draw(Actor* thisx, PlayState* play);
 
 void func_8098CB70(EnSi* this, PlayState* play);
 void func_8098CBDC(EnSi* this, PlayState* play);
+void func_8098CAD0(EnSi* this, PlayState* play);
+void func_8098CA20(EnSi* this, PlayState* play);
```

Comment:
I think the only forward declared function you need is `func_8098CBDC`, but if you do keep all the forward declarations, having them in descending/declaration order would be nice.

Reply:
You're right! How did you figure it out?

Reply:
It's a quirk of C, if you look at how all the functions call each other, `func_8098CB70` calls `func_8098CB70` before it is declared, so without a forward declaration the compiler will complain that it doesn't know what `func_8098CB70` is.